src/Model/GroundWaterFlow/submodules/gwf-sfr-transient.f90

```diff
@@ -0,0 +1,190 @@
+submodule(SfrModule) SfrModuleTransient
```

Comment:
Is this our first use of submodules in develop?  Is it necessary since there is an interface to sfr_calc_transient?

Reply:
No. There are `gwf-sfr-steady` and `gwf-sfr-constant` submodules already. The submodules were introduced after the last release. The interface is required for a submodule. The submodule can access variables in `SfrModule` as `this%variable`.